wpeverest/everest-forms
# Enhancement - Captcha field.

### All Submissions:


### Changes proposed in this Pull Request:
Previoulsy. there was no captcha field users have to got to form builder of security and anitspam setting to enable the captcha using check.This PR removed the previous setting and introduced  captcha field.


### How to test the changes in this Pull Request:

1. Drag and Drop the captcha field( reCaptcha, hCaptcha, Cloudflare Turnstile).
2. Preview the form and check all the captcha are working accordingly or not.

Important Note :- Check if the previously user has checked the recaptcha then automatically field should be dragged of that captcha and check if there any backward compatibility issue. 

### Types of changes:




### Other information:




### Changelog entry

> Enhancement - Captcha field.
